test/unit/breadcrumbs.ts

```diff
@@ -4,15 +4,13 @@ import Breadcrumbs from '../../src/breadcrumbs';
 import suite from './_suite';
 
 const QUERY = 'ballroom shoes';
-const STATE = { a: 'b' };
 
 suite('Breadcrumbs', ({ expect, spy, stub, itShouldBeConfigurable, itShouldHaveAlias }) => {
-  let querySelector: sinon.SinonStub;
   let breadcrumbs: Breadcrumbs;
 
   beforeEach(() => {
-    querySelector = stub(Selectors, 'query').returns(QUERY);
-    Breadcrumbs.prototype.flux = <any>{ store: { getState: () => STATE } };
+    Breadcrumbs.prototype.select = <any>spy(() => QUERY);
```

Comment:
we should save this as a reference `let select` for reference in tests rather than referencing the prototype